test/qa/lib/__init__.py

```diff
@@ -333,3 +333,23 @@ def get_virtual_countries() -> list[str]:
         countries.append(country)
 
     return countries
+
+
+class CommandExecutor:
```

Comment:
Is there a plan to use this across all the code in `qa/lib` or it's just for fileshare?